easybuild/easyconfigs/p/Perl/Perl-5.22.1-intel-2016a.eb

```diff
@@ -0,0 +1,909 @@
+name = 'Perl'
+version = '5.22.1'
+
+homepage = 'http://www.perl.org/'
+description = """Larry Wall's Practical Extraction and Report Language"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+source_urls = ['http://www.cpan.org/src/5.0']
+sources = [SOURCELOWER_TAR_GZ]
+
+exts_list = [
+    ('Config::General', '2.58', {
```

Comment:
why not? All others have this too. If you want updating, do it in a seperate PR.